### PR TITLE
Evany blog

### PR DESCRIPTION
**Pull request by IvanEvan**
Fixes # .

Make sure these boxes are checked before creating your pull request -- thank you!

- [x] Add Blog Header, including title, author, date.

e.g.

``` md
---
 title: "'Score Your Face Photo' a ML&Wechaty practice"
 date: 2017-09-18 09:00 +0800
 author: huyingxi
 ---
 > Author: [@huyingxi](https://github.com/huyingxi/wechaty_selfie) enjoying ML&Wechaty at BUPT
```

- [x] Keep all filenames & url as lowercase, and use `-` to connect words instead of space. e.g. `2017-10-06-wechat-pc-impactor` instead of `2017-10-06-WeChat PC Impactor`,
- [x] Embed the photo & video before publishing, save all external file to the blog `/download/2018` directory.
- [x] Add `<!--more-->` section to just show the abstract in blog homepage.
- [x] Add teaser image for the blog
- [x] Add your self as an author to the author data file at <https://github.com/wechaty/wechaty.github.io/blob/master/_data/authors.yml> with your GitHub username. And please also remember to reference yourself with the Github username in your blog post, so that we can have nice author information besides your blog after we published it!

[See more](https://github.com/wechaty/wechaty.github.io#contribute-guideline)

### CODE REVIEW

File: _posts/2020-06-15-daily-push-bot.md [hunk 1]
```diff
@@ -0,0 +1,151 @@
+---
+title: "资讯自动推送机器人（Daily push wechat robot）"
+author: IvanEvan
```
Comment (lijiarui) [inline _posts/2020-06-15-daily-push-bot.md]: This should same as the name in `author.yml`. So you should change to `Evany`

**Review by lijiarui — CHANGES_REQUESTED** [review]
Please change as I request and sign the CLA.

Then I can merge this PR.

File: _posts/2020-06-15-daily-push-bot.md (hunk 1 above)
Comment (IvanEvan) [reply _posts/2020-06-15-daily-push-bot.md]: Got it, But what should I do?
Start a new Pull request？

Comment (lijiarui) [reply _posts/2020-06-15-daily-push-bot.md]: nop, just continue to commit into this pr.

**Review by lijiarui — CHANGES_REQUESTED** [review]

**Review by lijiarui — APPROVED** [review]
Good job！

**Comment by lijiarui** [comment]
Congratulations! 🎉 🎉 🎉

You become a wechaty contributor successfully! 👏 👏 👏

Please:

- [ ]  Add my wechat: ruirui_0914 and send me this pr link. I will invite you into Wechaty Contributor Room
- [ ] Update your profile here: [Contributor List](https://github.com/wechaty/wechaty/wiki/Contributors), then other contributors will know you better!
- [ ]  Join wechaty org by clicking https://github.com/wechaty

image

**Comment by IvanEvan** [comment]
> Congratulations! 🎉 🎉 🎉
> 
> You become a wechaty contributor successfully! 👏 👏 👏
> 
> Please:
> 
> * [ ]   Add my wechat: ruirui_0914 and send me this pr link. I will invite you into Wechaty Contributor Room
> * [ ]  Update your profile here: [Contributor List](https://github.com/wechaty/wechaty/wiki/Contributors), then other contributors will know you better!
> * [ ]   Join wechaty org by clicking https://github.com/wechaty
> 
> image

Thx~~~

**Pull request merged**